src/streamlink_cli/argparser.py

```diff
@@ -481,6 +481,74 @@ def build_parser():
         This option will instead let the player decide when to exit.
         """
     )
+    player.add_argument(
+        "-t", "--title",
+        metavar="TITLE",
+        help="""
+        This option allows you to supply a title to be displayed in the 
+        title bar of the window that the video player is launched in.
+        
```

Comment:
could you remove the not used whitespace

`W293 blank line contains whitespace`

line

490
496
501
504
507
512
516
532
535
542
546